Jellyfin.Data/DayOfWeekHelper.cs

```diff
@@ -12,8 +12,8 @@ public static DayOfWeek[] GetDaysOfWeek(DynamicDayOfWeek day)
             return day switch
             {
                 DynamicDayOfWeek.Everyday => new[] { DayOfWeek.Sunday, DayOfWeek.Monday, DayOfWeek.Tuesday, DayOfWeek.Wednesday, DayOfWeek.Thursday, DayOfWeek.Friday, DayOfWeek.Saturday, DayOfWeek.Sunday },
-                DynamicDayOfWeek.Weekday => new[] { DayOfWeek.Sunday, DayOfWeek.Monday, DayOfWeek.Tuesday, DayOfWeek.Wednesday, DayOfWeek.Thursday, DayOfWeek.Friday },
```

Comment:
Sunday being a weekend or not is very culture dependent. Not sure if it should be hardcoded or not.